node/src/components/storage/lmdb_ext.rs

```diff
@@ -10,11 +10,18 @@
 //! Serialization errors are unified into a generic, type erased `std` error to allow for easy
 //! interchange of the serialization format if desired.
 
+use std::any::TypeId;
+
 use lmdb::{Database, RwTransaction, Transaction, WriteFlags};
 use serde::{de::DeserializeOwned, Serialize};
 use thiserror::Error;
 
-use casper_types::bytesrepr::{self, FromBytes, ToBytes};
+use casper_types::{
+    bytesrepr::{self, FromBytes, ToBytes},
+    system::auction::UnbondingPurse,
+};
+
+const UNBONDING_PURSE_V2_MAGIC_BYTES: &[u8] = &[121, 17, 133, 179, 91, 63, 69, 222];
```

Comment:
Is there a reason why a serialized V1 purse can never start with those bytes?

Reply:
Unfortunately not.
The `UnbondingPurse` starts with an `URef`, which starts with an `URefAddr`, which is just a `[u8; 32]` that could be anything.
I used 8 magic bytes as an effort to minimize the chances for a collision.